WordPress/src/jalapeno/res/drawable/app_icon_foreground.xml

```diff
@@ -0,0 +1,23 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="108dp"
+    android:width="108dp"
+    android:viewportHeight="108.0"
+    android:viewportWidth="108.0" >
+
+    <path
+        android:fillColor="#ff77b255"
+        android:pathData="M83,54c0-16-13-29-29-29c-1.8,0-3.6,0.2-5.3,0.5c-0.1,0.2-0.2,0.3-0.4,0.4c0,0-8,13.7,0,32 c2.2,4.9,12.4,17.9,15.8,23.2C75.1,77,83,66.4,83,54z" >
+    </path>
+
+    <path
+        android:fillColor="#fff4900c"
+        android:pathData="M48.3,25.9c0.2-0.1,0.3-0.3,0.4-0.4C35.3,28,25,39.9,25,54c0,16,13,29,29,29c3.5,0,7-0.6,10.1-1.8 c-3.4-5.4-13.6-18.4-15.8-23.2C40.3,39.6,48.3,25.9,48.3,25.9z" >
+    </path>
```

Comment:
```suggestion
    <path
        android:fillColor="#ffdce775"
        android:pathData="M83,54c0,16-13,29-29,29S25,70,25,54s13-29,29-29S83,38,83,54z" >
    </path>
```

In order to differentiate `jalapeno` and `wasabi` flavors, I would change the `jalapeno` foreground to `#dce775` to look like a jalapeno.